### DISCUSSION TITLE
What is Poetry's release schedule?

### DISCUSSION THREAD

**Discussion by tomasz-sodzawiczny** [python-poetry/poetry#7868] [Q&A] [answered]
What is the release schedule for poetry? I see that 1.4.2 was released over a month ago, and there have been some PRs merged since, with features I'm really looking forward to. Where can I track when the next release is going to be out?

**Comment by finswimmer** [comment, marked as answer]
Hey @tomasz-sodzawiczny,

we don't have a fixed release cycle. It depends on how much time we have.

Poetry 1.5 is not far away. You can follow #7853 for it.

fin swimmer
